### ISSUE TITLE
Node menu will not work

### ISSUE THREAD

**Issue by pushpender98** [BALKANGraph/OrgChartJS#435]
Hi

If I make the settings like `this.nodeMouseClick  = OrgChart.action.pan`

and I have used 
```
chart.nodeMenuUI.on('show', function (sender, args) {
        // code
    });
```
Then on clicking on the node menu the node menu will not work.

**Comment by ZornitsaSerbezova** [comment]
Thank you for reporting this issue. We will log it ad will try to fix it when we are able.

**Comment by ZornitsaPesheva** [comment]
Fixed in 7.10.40
